cli/commands/terraform/creds/providers/externalcmd/provider.go

```diff
@@ -43,6 +43,7 @@ func (provider *Provider) GetCredentials(ctx context.Context) (*providers.Creden
 		args = parts[1:]
 	}
 
+	provider.terragruntOptions.Logger.Infof("Executing %s to obtain Terragrunt credentials", provider.Name())
```

Comment:
Open to any suggestions for wording / format - my goal here is to help explain why a particular command started to execute when viewing log output with many steps.

Reply:
Everything is great, thanks for your contribution!